relay-general/src/protocol/measurements.rs

```diff
@@ -115,6 +119,12 @@ fn test_measurements_serialization() {
     "measurements": {
       "": {
         "err": [
+          [
+            "invalid_data",
+            {
+              "reason": "measurement name '' can contain only characters a-z0-9.-_"
```

Comment:
Strictly speaking, this message is not correct.

Reply:
I updated the message to specify that it cannot be empty.